src/generate.jl

```diff
@@ -0,0 +1,74 @@
+function new(pkg::String; minimal::Bool=true)
+    minimal == true || cmderror("only minimal package generation implemented")
+    isdir(pkg) && cmderror("$(abspath(pkg)) already exists")
+    project(pkg)
+    entrypoint(pkg)
+end
+
+function genfile(f::Function, pkg::String, file::String)
+    path = joinpath(pkg, file)
+    @info "Generating $(pkg)/$(file)"
+    mkpath(dirname(path))
+    open(f, path, "w")
+    return
+end
+
+function project(pkg::String)
+    # Find a name + email
+    name = email = nothing
+    dot_gitconfig = joinpath(homedir(), ".gitconfig")
+    if isfile(dot_gitconfig)
+        content = read(dot_gitconfig, String)
+        m = match(r"name = (.*?)\n", content)
+        m != nothing && (name = m.captures[1])
+        m = match(r"email = (.*?)\n", content)
+        m != nothing && (email = m.captures[1])
+    end
```

Comment:
Why parse `.gitconfig` for this? Why not use `libgit2` or `git` to get this info?

Reply:
I'll take it from LibGit2 instead.